### ISSUE TITLE
Are Entity's traveling with crafts configurable?

### ISSUE THREAD

**Issue by pistols12** [msummers123/Movecraft-3#237]
we have no monsters on our server, and people desperately want to transport animals, is there a way to configure if entity's move with crafts or not? as of now they do not.

**Comment by FrozenBeard** [comment]
I agree that entity moving should be re-added there is a fork of movecraft with this working, please implement their public changes to re-allow entity movement with crafts.

**Comment by TylerS1066** [comment]
The issue with transporting animals was that it wipes their movement AI, so they become dumb statues.  Yarro then decided to just disable it fully, I'm sure it would be easy to re-add a config for it.

**Comment by eirikh1996** [comment]
Shouldn't it in some way be possible to restore the movement AI after transport?

**Comment by pistols12** [comment]
bump
